Nodejs/Product/Analysis/Analysis/AnalysisLimits.cs

```diff
@@ -34,6 +34,17 @@ internal class AnalysisLimits {
             MaxObjectLiteralProperties = 50;
             MaxObjectKeysTypes = 5;
             MaxMergeTypes = 5;
+            NestedModulesLimit = 4;
```

Comment:
Do we really want to mess with the high analysis limits here? How did you decide on 4?